store.go

```diff
@@ -1745,7 +1804,7 @@ func (s *store) ImageSize(id string) (int64, error) {
 }
 
 func (s *store) ContainerSize(id string) (int64, error) {
-	lstore, err := s.LayerStore()
+	lstore, err := s.ROLayerStore()
```

Comment:
Pretty sure this needs to be a writeable store, because it uses `DiffSize()`.  We probably don't need to bother with any read-only stores in here, since we should only be finding container read-write layers in the read-write store, which is the only place we can create them.

Reply:
I changed it to `RWLayerStore()` and removed the ROLayers. PTAL